export const start = async () => {
  return {};
};

(async () => {
  await start();
})().catch(console.error);
